hack/openshift.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2017 Red Hat, Inc.
+#
+
+#fist make sure externalIPNetworkCIDRs is set to ["0.0.0.0/0"] in master config
+# refer to https://access.redhat.com/solutions/2464791
+# this is needed for the services that declare use of external ips
+
+PROJECT="default"
+oc create sa kubevirt-controller
+oc adm policy add-cluster-role-to-user cluster-admin -z kubevirt-controller
+oc patch deployment/virt-controller --patch '{"spec":{"template":{"spec":{"serviceAccountName": "kubevirt-controller"}}}}'
```

Comment:
Would it be possible, to give our kubevirt users which we create as part of rbac the right permissions, instead of changing the depolyments?